dev.overlayed.Overlayed.yaml

```diff
@@ -0,0 +1,81 @@
+id: dev.overlayed.Overlayed
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk//46
```

Comment:
```suggestion
sdk: org.gnome.Sdk
```

Reply:
I there a reason I have to omit this? If I do this it's going to bust my  compile cache of my linux test device.

Reply:
No need to build again, you don't need to specify the version twice.